<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Polen Tespit ve Sınıflandırma Programı v1.1</string>
  </property>
  <property name="toolTip">
   <string>Kullanmaya Başlamak için Görüntü yükleyiniz.</string>
  </property>
  <property name="statusTip">
   <string>Mikroskop görüntülerinden yapay zeka ile polen sınıflandırma aracı</string>
  </property>
  <property name="whatsThis">
   <string>Mikroskop görüntülerinden yapay zeka ile polen sınıflandırma aracı</string>
  </property>
  <property name="iconSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
     </font>
    </property>
    <property name="accessibleDescription">
     <string>Mikroskop görüntülerinden yapay zeka ile polen sınıflandırma aracı</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="accessibleName">
      <string/>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Ana Ekran</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
       </font>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
          </font>
         </property>
         <property name="text">
          <string>RESİM YÜKLE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="classifyButton">
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
          </font>
         </property>
         <property name="text">
          <string>SINIFLANDIR!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>331</width>
        <height>281</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
       </font>
      </property>
      <property name="title">
       <string>Polen</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>311</width>
         <height>231</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Reference Sans Serif</family>
         <pointsize>7</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Başlamak için polen görüntüsü yükleyiniz...</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>24</x>
         <y>260</y>
         <width>281</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>0</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>6</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Temizle</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>281</width>
        <height>321</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
       </font>
      </property>
      <property name="title">
       <string>Sınıflandırma Sonucu</string>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>271</width>
         <height>111</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>271</width>
         <height>181</height>
        </rect>
       </property>
       <property name="title">
        <string>Referans Polen</string>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>241</width>
          <height>151</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Reference Sans Serif</family>
          <pointsize>7</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Referans alınan veya polen görüntüsü</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Yardım ve Bilgi</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>541</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Yapay Zeka destekli Polen Tespit Programi v1.1 - Mehmet Cagri Aksoy 03.2024</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>441</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Mikroskop görüntülerinden yapay zeka ile polen sınıflandırma aracı</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
